docs/unified/novacustom/camera-switch.md

```diff
@@ -0,0 +1,9 @@
+# Camera switch setting
+
+Novacustom laptops have the option to disable the webcam completely. After
```

Comment:
```suggestion
NovaCustom laptops have the option to disable the webcam completely. After
```

Reply:
fixed in 42563eee50bae817d0b20418e1b73d34775acafc